public_suffix_list.dat

```diff
@@ -15218,6 +15218,10 @@ streamlitapp.com
 // Submitted by Ian Streeter <ian@snowplowanalytics.com>
 try-snowplow.com
 
+// Software Consulting Michal Zalewski : https://www.mafelo.com
+// Submitted by Michal Zalewski <michal.zalewski@mafelo.com>
```

Comment:
Please use a non-personal email address.

Reply:
I changed email to our email group